lib/client.js

```diff
@@ -23,6 +23,15 @@ var _defaultOptions = {
     certificate:  process.env.TASKCLUSTER_CERTIFICATE
   },
 
+  timeout: 30 * 1000, // 30 seconds
+
+  // Http agent options (https://nodejs.org/api/http.html#http_new_agent_options)
+  agent: {
+    keepAlive: true,
+    // XXX: This number is a random guess...
+    maxSockets: 256
```

Comment:
Well... It depends on the use-case... I guess, if you're a server that does stuff for people, you have a high constant load, and high number of connection is probably good.

If you are an end-user running locally, and just occasionally submitting a few kilo tasks, then you'll incur a much more steady load by only using `5` sockets. Also keeping alive a lot of sockets would be bad if you don't use them. Which you wouldn't do if you're a end-user running `taskcluster run-graph` locally, and that command decides to listen for events after submitting tasks.